### ISSUE TITLE
Resolve CodeQL alerts 36 and 37

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#5660]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to analyze [CodeQL query alert 36](https://github.com/hackforla/website/security/code-scanning/36) and [CodeQL query alert 37](https://github.com/hackforla/website/security/code-scanning/37), then either recommend dismissal of the alerts or update the code files to resolve the alerts  

### Action Items
- [ ] DO NOT DISMISS ANY ALERTS.  Dismissal of alerts should be done by dev leads only after review of the recommendation
- [ ] Examine the details provided in the alert pages linked in the following two items.   Click "See More" to view Recommendations, Examples and References.  
- [ ] https://github.com/hackforla/website/security/code-scanning/36
- [ ] https://github.com/hackforla/website/security/code-scanning/37
- [ ] In a comment in this issue, add your analysis and recommendations.  The recommendation can be one of the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, or `update code`.  An example of a `false positive` is a report of a JavaScript syntax error that is caused by markdown or liquid symbols such as `---` or `{%`  
- [ ] If the recommendation is to dismiss the alert, apply the label `ready for dev lead` then move the issue to `Questions/In Review`
- [ ] If the recommendation is to update code:
   - [ ] create an issue branch and proceed with the code update
   - [ ] test using docker to ensure that there are no changes to any affected webpage(s)
   - [ ] proceed with pull request in the usual manner 

### For merge team/dev lead
- [ ] If recommendation to dismiss is approved, dismiss the alert with a comment, then close the issue as completed.
- [ ] If recommendation to update code is approved, move the issue to "In Progress", remove "ready for dev lead" label and notify assignee to proceed
- [ ] In either case when this issue is closed please check off the dependency (under "Analysis Issues") in #5060.  If all analysis issues are closed, close #5060 as completed.  
  
### Resources/Instructions
- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- This issue is part of #5060

**Comment by roslynwythe** [comment]
- Closing as not planned because CodeQL alerts 36 and 37 were resolved via #6152
